infra/scripts/node.toml

```diff
@@ -1,19 +1,44 @@
 ########################################################################################################################
 # Node helpers
 ########################################################################################################################
+
+[tasks.bootstrap]
+env = { "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_DIR}/bootstrap/cometbft", "CMT_CONTAINER_NAME" = "cometbft-${NETWORK_NAME}-bootstrap", "CMT_HOST_PORT" = "26650"}
+run_task = "new-node"
+
+[tasks.bootstrap-id]
+env = { "CMT_CONTAINER_NAME" = "cometbft-${NETWORK_NAME}-bootstrap", "CMT_HOST_PORT" = "26650"}
+run_task = "cometbft-node-id"
+
+[tasks.bootstrap-down]
+env = { "CMT_CONTAINER_NAME" = "cometbft-${NETWORK_NAME}-bootstrap" }
+run_task = "node-down"
+
+[tasks.bootstrap-mkdir]
+script="""
+echo creating directories: $BASE_DIR $FM_DIR $CMT_DIR
+mkdir -p $BASE_DIR
+
+mkdir -p $CMT_DIR
+"""
+
 [tasks.node-down]
 dependencies = [
     "cometbft-stop",
     "cometbft-rm"
 ]
 
+
 [tasks.new-node]
 dependencies = [
     "node-down",
-    "node-init",
+    "bootstrap-mkdir",
```

Comment:
why is it called `bootstrap-mkdir` and is used in `new-node`?

Reply:
I thought `new-node` was being used by some other script. It turns out it is not. I just wanted to do `node-mkdir` without creating a fendermint directory, this is why I wrote a new action for the bootstrap, but I realized it is not even needed because no one else is using it. I updated it accordingly.